Tools/CISupport/ews-build/steps.py

```diff
@@ -113,6 +113,12 @@ class GitHub(object):
     REQUEST_MERGE_QUEUE_LABEL = 'request-merge-queue'
     BLOCKED_LABEL = 'merging-blocked'
     SKIP_EWS_LABEL = 'skip-ews'
+    NO_FAILURE_LIMITS = 'no-failure-limits'
+    LABELS = [
```

Comment:
nit: We could put all the label constants in an [`Enum`](https://docs.python.org/3/library/enum.html?highlight=enum#module-enum) and then we wouldn't need to maintain a separate listing.

Reply:
I think in this case, that would make things more confusing, because this is part of the `GitHub` class, not the `GitHubLabels` class.